docs/asciidoc/static/codec.asciidoc

```diff
@@ -0,0 +1,13 @@
+:register_method:	true
+:encode_method:		true
+:decode_method:		true
+:plugintype:    	codec
+:pluginclass:   	Codecs
+:pluginname:    	example
+:pluginnamecap: 	Example
+
+:getstarted: Let's step through creating a {plugintype} plugin using the https://github.com/logstash-plugins/logstash-codec-example/[example {plugintype} plugin].
+
```

Comment:
Can we open this link in another tab/window perhaps, so the reader doesn't have to navigate back and forth?